packages.yml

```diff
@@ -6,3 +6,5 @@ packages:
   warn-unpinned: false
 - package: dbt-labs/spark_utils
   version: [">=0.3.0", "<0.4.0"]
+- package: calogica/dbt_date
+  version: [">=0.7.0", "<0.8.0"]
```

Comment:
I noticed calogica isn't compatible with Redshift, could this cause any issues with the macro in the future?

Reply:
it's compatible with postgres so it _should_ work with redshift, but it theoretically could!! in that case we'd open a PR ourselves